Dockerfile

```diff
@@ -41,31 +43,38 @@ RUN python3.12 get-pip.py
 RUN apt -y autoremove
 
 # Add user so we don't need --no-sandbox.
-RUN groupadd --system pptruser && \
-    useradd --system --create-home --gid pptruser pptruser && \
-    mkdir --parents /usr/src/app
-RUN chown --recursive pptruser:pptruser /usr/src/app
+RUN groupadd --system sitespeedio && \
+    useradd --system --create-home --gid sitespeedio sitespeedio && \
+    mkdir --parents /usr/src/runner
+RUN chown --recursive sitespeedio:sitespeedio /usr/src/runner
 
-WORKDIR /usr/src/app
+WORKDIR /usr/src/runner
 
 RUN echo 'ALL ALL=NOPASSWD: /usr/sbin/tc, /usr/sbin/route, /usr/sbin/ip' > /etc/sudoers.d/tc
 
 RUN npm install -g node-gyp puppeteer
 
 # If own config.py exists it will overwrite the SAMPLE
-COPY . /usr/src/app
+COPY . /usr/src/runner
 
-RUN chown --recursive pptruser:pptruser /usr/src/app
+RUN chown --recursive sitespeedio:sitespeedio /usr/src/runner
 
 # Run everything after as non-privileged user.
-USER pptruser
+USER sitespeedio
 
 RUN npm install
 
+WORKDIR /usr/src/runner/node_modules/sitespeed.io/
```

Comment:
@krompaco why is this needed?

Reply:
I tried multiple times and got missing file error unless I did a separate install like this. Could be something with cached folders in the new image.